types/sinonjs__fake-timers/index.d.ts

```diff
@@ -8,368 +8,380 @@
-     *
-     * @param id   Timer ID or object.
-     */
-    clearInterval: (id: TTimerId) => void;
-
-    /**
-     * Schedules the callback to be fired once 0 milliseconds have ticked by.
-     *
-     * @param callback   Callback to be fired.
-     * @remarks You'll still have to call clock.tick() for the callback to fire.
-     * @remarks If called during a tick the callback won't fire until 1 millisecond has ticked by.
-     */
-    setImmediate: (callback: () => void) => TTimerId;
-
-    /**
-     * Clears a timer, as long as it was created using setImmediate.
-     *
-     * @param id   Timer ID or object.
-     */
-    clearImmediate: (id: TTimerId) => void;
-
-    /**
-     * Implements the Date object but using this clock to provide the correct time.
-     */
-    Date: typeof Date;
-}
-
-/**
- * Timer object used in node.
- */
-export interface NodeTimer {
-    /**
-     * Stub method call. Does nothing.
-     */
-    ref(): void;
-
-    /**
-     * Stub method call. Does nothing.
-     */
-    unref(): void;
-}
-
-/**
- * Timer identifier for clock scheduling.
- */
-export type TimerId = number | NodeTimer;
-
-/**
- * Controls the flow of time.
- */
-export interface FakeClock<TTimerId extends TimerId> extends GlobalTimers<TTimerId> {
-    /**
-     * Current clock time.
-     */
-    now: number;
-
-    /**
-     * Don't know what this prop is for, but it was included in the clocks that `createClock` or
-     * `install` return (it is never used in the code, for now).
-     */
-    timeouts: {};
-
-    /**
-     * Maximum number of timers that will be run when calling runAll().
-     */
-    loopLimit: number;
-
-    /**
-     * Schedule callback to run in the next animation frame.
-     *
-     * @param callback   Callback to be fired.
-     * @returns Request id.
-     */
-    requestAnimationFrame: (callback: (time: number) => void) => TTimerId;
-
-    /**
-     * Cancel animation frame request.
-     *
-     * @param id   The id returned from requestAnimationFrame method.
-     */
-    cancelAnimationFrame: (id: TTimerId) => void;
-
-    /**
-     * Queues the callback to be fired during idle periods to perform background and low priority work on the main event loop.
-     *
-     * @param callback   Callback to be fired.
-     * @param timeout   The maximum number of ticks before the callback must be fired.
-     * @remarks Callbacks which have a timeout option will be fired no later than time in milliseconds.
-     */
-    requestIdleCallback: (callback: () => void, timeout?: number) => TTimerId;
-
-    /**
-     * Clears a timer, as long as it was created using requestIdleCallback.
-     *
-     * @param id   Timer ID or object.
-     */
-    cancelIdleCallback: (id: TTimerId) => void;
-
-    /**
-     * Get the number of waiting timers.
-     *
-     * @returns number of waiting timers.
-     */
-    countTimers: () => number;
+// tslint:disable-next-line no-single-declare-module
+declare module '@sinonjs/fake-timers' {
```

Comment:
what happened here? the exports were right before, we shouldn't have to explicitly declare a module

Reply:
you're right. i thought root cause was due to not declaring the ambient module, but it was just the tsconfig path map issue. will revert this change.

Reply:
change pushed up, will squash again once things are fully approved